packages/headless/src/controllers/facets/facet/headless-facet.ts

```diff
@@ -96,15 +96,15 @@ export function buildFacet(
     ...controller,
     facetSearch: createFacetSearch(),
     /**
-     * Selects (deselects) the passed value if unselected (selected).
-     * @param selection The facet value to select or deselect.
+     * Toggles the passed value.
+     * @param selection (FacetValue) The facet value to select or deselect.
      */
     toggleSelect: (selection: FacetValue) =>
       dispatch(executeToggleFacetSelect({facetId: options.facetId, selection})),
     /**
      * Returns `true` if the passed facet value is selected and `false` otherwise.
```

Comment:
Since we already have a returns annotations, perhaps we could be a little less wordy with this description. What would think of just:
Checks whether the passed(specified?) facet value is selected.